api/src/test/java/io/opentelemetry/api/OpenTelemetryTest.java

```diff
@@ -59,10 +57,12 @@ void after() {
 
   @Test
   void testDefault() {
-    assertThat(OpenTelemetry.getGlobalTracerProvider()).isInstanceOf(DefaultTracerProvider.class);
+    assertThat(OpenTelemetry.getGlobalTracerProvider().getClass().getSimpleName())
+        .isEqualTo("DefaultTracerProvider");
```

Comment:
I wonder how much value these tests are really providing.

Reply:
I think they make sure we don't actually add a dependency on the SDK from the API and don't know if any of our other tests do. So that's probably good value.